webapp/__init__.py

```diff
@@ -0,0 +1,10 @@
+from webapp.app import app
+
+
+# this is template filter that will sort a provided list of dicts
+# by a key in order of values that are provided in `ordered_list`
+@app.template_filter()
+def sort_by_key_and_ordered_list(list_to_sort, obj_key, ordered_list):
```

Comment:
I would not put this in the `__init__.py` nothing wrong with it, but we use `handlers.py` to register filters (ex: Line 181) or other app contexts, then we define the function inside `context.py`. I think it's nicer to keep things consistent.

Reply:
okay, I was searching for `@app.template_filter()` and couldn't find it anywhere, now I see how it's done. Thanks, Albert! Fixed.